src/OpenTelemetry.Exporter.Geneva/Internal/MsgPack/MsgPackLogExporter.cs

```diff
@@ -417,22 +417,32 @@ internal ArraySegment<byte> SerializeLogRecord(LogRecord logRecord)
 
             cursor = MessagePackSerializer.SerializeAsciiString(buffer, cursor, "env_ex_msg");
             cursor = MessagePackSerializer.SerializeUnicodeString(buffer, cursor, logRecord.Exception.Message);
-            cntFields += 1;
-
+            cntFields += 1;
+
+            // The current approach relies on the existing trim
+            // capabilities which trims string in excess of STRING_SIZE_LIMIT_CHAR_COUNT
+            // TODO: Revisit this:
+            // 1. Trim it off based on how much more bytes are available
+            // before running out of limit instead of STRING_SIZE_LIMIT_CHAR_COUNT.
+            // 2. Trim smarter, by trimming the middle of stack, an
+            // keep top and bottom.
             if (this.exportExceptionStack == ExceptionStackExportMode.ExportAsString)
             {
-                // The current approach relies on the existing trim
-                // capabilities which trims string in excess of STRING_SIZE_LIMIT_CHAR_COUNT
-                // TODO: Revisit this:
-                // 1. Trim it off based on how much more bytes are available
-                // before running out of limit instead of STRING_SIZE_LIMIT_CHAR_COUNT.
-                // 2. Trim smarter, by trimming the middle of stack, an
-                // keep top and bottom.
                 var exceptionStack = logRecord.Exception.ToInvariantString();
                 cursor = MessagePackSerializer.SerializeAsciiString(buffer, cursor, "env_ex_stack");
                 cursor = MessagePackSerializer.SerializeUnicodeString(buffer, cursor, exceptionStack);
                 cntFields += 1;
             }
+            else if (this.exportExceptionStack == ExceptionStackExportMode.ExportAsStackTraceString)
+            {
+                var exceptionStack = logRecord.Exception.StackTrace;
+                if (exceptionStack != null)
```

Comment:
Is there a reason why we check here only for null, while on TLD we use `string.IsNullOrEmpty`?

Reply:
good catch! Don't think we need either checks and serialize them as-is, as already done.